src/state.rs

```diff
@@ -161,13 +161,17 @@ impl BarState {
     }
 
     pub(crate) fn suspend<F: FnOnce() -> R, R>(&mut self, now: Instant, f: F) -> R {
-        if let Some(drawable) = self.draw_target.drawable(true, now) {
-            let _ = drawable.clear();
-        }
+        if let Some((state, _)) = self.draw_target.remote() {
+            state.write().unwrap().suspend(f, now)
```

Comment:
Style nit: please return from this `if`, so that we don't need to have the rest of the function in the `else` block (reducing rightward drift).

Reply:
Alright Changed it